mathics/builtin/evaluation.py

```diff
@@ -338,3 +335,26 @@ def eval(self, evaluation: Evaluation, n):
         if isinstance(n, Integer):
             exitcode = n.get_int_value()
         raise SystemExit(exitcode)
+
+
+class Exit(Quit):
+        """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/Quit.html</url>
```

Comment:
Should  all uses of Quit be Exit in the doc here?

Reply:
This is s duplicate of the docstring for Quit. Quit can be removed here, and "Exit" in the other docstring copy. It is okay however to mention that Quit and Exit are the same thing (if they are) or mention what the differences are if they differ somehow.